clippy_lints/src/utils/mod.rs

```diff
@@ -138,13 +215,8 @@ pub fn match_def_path(tcx: TyCtxt<'_, '_, '_>, def_id: DefId, path: &[&str]) ->
 ///     // The given `def_id` is that of an `Option` type
 /// };
 /// ```
-pub fn get_def_path(tcx: TyCtxt<'_, '_, '_>, def_id: DefId) -> Vec<&'static str> {
```

Comment:
@Zoxc How did this work? Surely if I can get a `&'static str` from an interner, I can pass it to another thread?

Reply:
@nnethercote was cheeky and added a public method to `LocalInternedString` which returns a `&'static str`